### PR TITLE
Fix the group access to the hypershift-toolkit namespace

### PR DESCRIPTION
**Pull request by csrwng**

### CODE REVIEW

File: clusters/app.ci/hypershift-toolkit/rbac.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
 subjects:
 - kind: Group
   apiGroup: rbac.authorization.k8s.io
-  name: hypershift-admin
+  name: hypershift-team
```
Comment (hongkailiu) [inline clusters/app.ci/hypershift-toolkit/rbac.yaml:10]: This change has to be aligned with 

https://github.com/openshift/release/blob/0ff38126443dd0df982af89264081a13c31d401d/core-services/sync-rover-groups/_config.yaml#L20-L23

Comment (csrwng) [reply clusters/app.ci/hypershift-toolkit/rbac.yaml:10]: ahh, so we need `hypershift-pool-admins` will switch

Comment (hongkailiu) [reply clusters/app.ci/hypershift-toolkit/rbac.yaml]: Could you please also add `app.ci` into the `clusters` list in the config file?

Comment (csrwng) [reply clusters/app.ci/hypershift-toolkit/rbac.yaml]: done

**Review by hongkailiu — APPROVED** [review]
Interesting

**Pull request merged**